src/app/pages/ArticlePage/ArticlePage.tsx

```diff
@@ -127,6 +130,8 @@ const ArticlePage = ({ pageData }: { pageData: Article }) => {
     palette: { GREY_2, WHITE },
   } = useTheme();
 
+  const experimentVariant = useOptimizelyMvtVariation('top_bar_oj_experiment');
```

Comment:
We'll need another variable to ensure that `null || 'off'` is ignored and won't add in the Optimizely components below or the `atiData` property.

Something like:
```
const experimentVariant = useOptimizelyMvtVariation('top_bar_oj_experiment');

const shouldRunExperiment = experimentVariant && experimentVariant !== 'off';

```

Then use that variable for the checks to add the Optimizely stuff.